### PR TITLE
Release/v1.3.0 pso

### PR DESCRIPTION
**Pull request by omenUchiha**
## Description

**Bugs**
- prod Signoff Issues

**What kind of change does this PR introduce?** (check at least one)

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Hotfix (fixes a breaking changes)
- [ ] New Feature (non-breaking change which adds functionality)
- [ ] Other, please describe:

### Author Checklist

- [x] Assignee added
- [x] Reviewer added
- [x] Description added
- [ ] Unit test cases added

### Reviewer Checklist

- [ ] PR title starts with Jira ticket number prefix
- [ ] If hotfix, Add Hotfix as prefix in PR title
- [ ] Label added
- [ ] Commit messages follows standard format (added, changed, removed, code commented and chore)
- [ ] Description about the Feature/ Bugs added

### CODE REVIEW

File: package.json
Comment (ishaan1607) [inline package.json:1]: remove this file.

File: package-lock.json
Comment (ishaan1607) [inline package-lock.json:1]: remove this file

**Review by ishaan1607 — CHANGES_REQUESTED** [review]

File: package.json
Comment (omenUchiha) [reply package.json:1]: Done

File: package-lock.json
Comment (omenUchiha) [reply package-lock.json:1]: Done

**Review by ishaan1607 — APPROVED** [review]

**Pull request merged**